client/state/action-types.js

```diff
@@ -330,6 +330,11 @@ export const HELP_TICKET_CONFIGURATION_REQUEST_FAILURE =
 export const HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS =
 	'HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS';
 export const HISTORY_REPLACE = 'HISTORY_REPLACE';
+export const HOSTING_CREATE_SFTP_USER = 'HOSTING_CREATE_SFTP_USER';
+export const HOSTING_RECEIVE_SFTP_USER = 'HOSTING_RECEIVE_SFTP_USER';
+export const HOSTING_RECEIVE_SFTP_USER_ERROR = 'HOSTING_RECEIVE_SFTP_USER_ERROR';
+export const HOSTING_REQUEST_SFTP_USER = 'HOSTING_REQUEST_SFTP_USER';
+export const HOSTING_RESET_SFTP_PASSWORD = 'HOSTING_RESET_SFTP_PASSWORD';
```

Comment:
I guess I missed Miguels below too, but naming would be:
`HOSTING_REQUEST_SFTP_USER ` -> `HOSTING_SFTP_USER_REQUEST`
`HOSTING_RESET_SFTP_PASSWORD` -> `HOSTING_SFTP_USER_RESET_PASSWORD`
`HOSTING_RESTORE_DATABASE_PASSWORD ` -> `HOSTING_DATABASE_PASSWORD_RESTORE`